src/components/App.js

```diff
@@ -0,0 +1,37 @@
+import ButtonPannel from './ButtonPanel';
+import Display from './Display';
+import Button from './Button';
+
+function App() {
+  return (
+    <div className="App">
+      <div className="App__calculator">
+        <Display />
```

Comment:
- [ ] Rather than using `div className='App'` to wrap the calculator components consider using 
[React.Fragment](https://reactjs.org/docs/fragments.html#short-syntax) short syntax ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone2/app/ReactFragment.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896092/))

Reply:
Though I used the outer div to style the app. I have wrapped every element in a react Fragment